app/backend/src/couchers/notifications/render.py

```diff
@@ -847,5 +847,22 @@ def render_notification(user, notification) -> RenderedNotification:
             push_icon=urls.icon_url(),
             push_url=urls.app_link(),
         )
+    elif notification.topic_action.display == "general:new_blog_post":
+        title = f"New blog post: {data.title}"
+        return RenderedNotification(
+            email_subject=title,
+            email_preview=data.blurb,
+            email_template_name="new_blog_post",
+            email_template_args={
+                "title": data.title,
+                "blurb": data.blurb,
```

Comment:
Is the length of the blurb already determined somewhere else? If so we probably want to make this one shorter as on the blog page the blurb can be a paragraph.

Reply:
Hmm, it seems the character counts depends on the platform. I've added some random limits...